### PR TITLE
Website Rework

### LINKED ISSUE #102: Repetitive blog post images
Perhaps we should consider using other images here to break up the monotony? Perhaps using a consistent image for each project: `riskmetric` updates, `riskassessment` updates, & `event` updates could all have their own unique image and other misc updates could default to this pharmar logo image? This would help us visually separate them on first glance.

image

### LINKED ISSUE #105: Link minutes to github
We used to link the minutes on the webpage
https://www.pharmar.org/minutes/

Since we are having them now in github, we should probably link there: 
https://github.com/pharmaR/pharmaR/issues

### PR DESCRIPTION
**Pull request by jcabercrombie**
Website rework based on communications workstream discussions and comments from other workstream members. 

This PR also helps support issues in the repo and thus Closes #105 and Closes #102 and creates the foundation for #96.

### CODE REVIEW

**Comment by aclark02-arcus** [comment]
Hi @jcabercrombie, do you need someone to help review this PR? Let me know and I'd be happy to allot some time to it. Thanks!

**Comment by jcabercrombie** [comment]
Hi @AARON-CLARK - I would love a set of eyes on this once it's ready. I've been swamped with some deliverables this week and have not got around to fixing the deployment failures you see in this PR. I'll let you know as soon as I've fixed the issue. Thank you!

**Comment by dgkf** [comment]
Just one thing to maybe clean up before this goes to prod, we have this page that looks like it was just put in for testing:
https://deploy-preview-109--condescending-lewin-a2fc51.netlify.app/posts/case-studies/case-studies-post-test/

**Comment by jcabercrombie** [comment]
Thank you, @dgkf! It is in place so that @anujadas185 has a template to go off of for how to set up a case studies post, but we reviewed how to create a post on a call a couple of weeks ago so I should be free to remove it. I'll look into removing this today.

**Review by aclark02-arcus — CHANGES_REQUESTED** [review]
Hi @jcabercrombie,

Thanks for the walkthrough on Tuesday. Overall, it looks amazing. Light years away from where it was previously - a huge improvement. Here is my review of the [site](https://deploy-preview-109--condescending-lewin-a2fc51.netlify.app/):

### What I love

1. the landing page and carousel are beautiful

2. The flow seems more intuitive... that is navigating around for stuff I'm know exists and am trying to find.

3. Love the new "Posts" format, giving that dark blue background for the thumbnails.

4. These watermark images (below) are beautiful. Kudos to whoever put those together! However, I did want to ask what purpose they serve? Are they just to fill space since the `About Us` and `Validation` tabs have subheadings and sub-lists?

    image


### Small stuff
1. PSI logo not rendering on home page
 
    image

4. Looks like the form was removed from this page, so maybe remove this text? 
    
    image

    Here is a link to the form on the "old site" (aka, the current site).

    image


5. I know your team didn't edit the pre-existing content most pages, but this should really state "{riskassessment} application" instead of "Risk Assessment Application".

    image

6. Should this be updated to say "Posts" instead of "News" since as you mentioned, you're switching to a new naming convention for these threads?

    image

7. Consider changing "Background" to perhaps "Origins" here? Background is such a broad word. Just a suggestion. Maybe "background" is more appropriate. Note that it's also in the "About Us" dropdown list.

    image

8. Resources > Minutes > Repositories Working Group > [GitHub link](https://github.com/RConsortium/r-repositories-wg/tree/main/minutes)... this leads users to the R Consortium's Repositories group GitHub repo, which is the wrong repo. R Consortium has a working that is separate from the [R Validation Hub's Regulatory R working Group](https://github.com/pharmaR/regulatory-r-repo-wg). Not sure if they post their minutes anywhere. 

Another note on that: we (`riskassessment` workstream) doesn't post our minutes anywhere either, but if we are required to, we will. I usually just scratch down some loose notes every meeting, but don't publish them.

9. I know things are still in a quasi-draft mode, but wanted to point out that this post still has a "test" tag on it. Not sure if you want to rename that before going live?

    image


### Optional: use `{bslib}` to make cards

Inspired by your "Members & Workstreams" page & after learning we probably shouldn't prioritize company involvement (and logos) but more individual involvement on our workstreams, [I initiated a PR](https://github.com/pharmaR/riskassessment/pull/780#issuecomment-2117677442) on on the `riskassessment` app to update our "About" page. I first checked out your approach, but saw it relied heavily on `cowplot`. Since we didn't want to add another dependency into the app, we found that `bslib` (an existing dependency) does a good job creating custom cards that can filled with any content. Here is a snapshot of my progress so far:
    
image

Anyway, just an option. I think the cards developed in this website PR look really good as-is.

Along those lines, do you want me (or other core-contributors) to the `riskassessment` repo to add  their info here? There are 2 other "core" contributors that have been with the project for several years, so I'd say they're worthy of getting their mug shot published here. Any requirements on photos?

### In conclusion
It look great! I think after this get's merged, we can spend some time time updating some of the older content on some of the legacy pages that are sticking around.

**Comment by jcabercrombie** [comment]
Hi @aclark02-arcus! Thank you so much for the feedback and for taking the time to identify all the small things worth changing before going live. I've gone through each of your comments and have the following to share:

_Re: What I Love_
I am happy to hear things feel more intuitive and have a greater visual appeal—those were the main goals for this revamp! I threw the "watermark" images together to take up space so the drop-down menu for our main sections could look more fun. They're just to take up space as you said, as the Hugo theme we use allows for these larger drop-downs and I wanted to take advantage of some more visual variation the theme offers. In the future, we can replace these images with anything else if we see fit.

_Re: Small stuff_
1. I saw this as well when the website first rendered on GitHub. It works fine when rendering the site locally, so I'm not quite sure what's happening. For now, it is something we can look into after merging this PR.
2. I took a look at why this may have been removed, as there were no changes made to the contact.md file. This could have been an update with the theme. I can look into this more after the first deployment. I believe it had been going to an old R Validation Hub member and needed a revamp anyway based on this issue (#104) from Doug, so maybe its removal for now is fine.
3. I made the update for how riskassessment is displayed so that it's represented how you requested.
4. The incorporation of blog-type posts on the site is very interesting in terms of how the website uses folders to render. Basically, in our website directory, ~/posts/news and ~/posts/case-studies exist to allow me to separate them into their own URLs for the site. Any folder in the posts folder then becomes its own category (ex: your screenshot in 9) and I had to learn by trial-and-error how things work. "Posts" was the overarching name for what used to be "blogs." I think maintaining it as "News" makes sense since they're our "news" updates, but if over time we stratify our types of content more they can become their own folders in the post folder. I'm also open to renaming "News" to something else if you think a better category name exists. I hope this helps explain things! Let me know if you have more questions on this.
5. I love the word "origins!" I had struggled with naming this page too and went through a bunch of iterations. Ultimately it does talk about where we came from, so origins is more specific. I've made this change in my most recent commits. Thank you!
6. I just linked what Joe had provided during the last executive meeting, but for now I removed it since it does link to somewhere else. Thanks for the catch here! I'll ask Doug if he knows which groups create minutes or not. I know Joe had advocated for transparency and even deemed this minutes page one of our most important, so I wonder if beginning more formal notes for riskassessment would be a step in the right direction! Maybe a topic to bring up at the next exec meeting.
9. Thanks for pointing this out! The placeholder here was so that Anuja had an example to go off of when she creates the couple/few entries for case studies. Ideally, the case studies would be updated before our initial deployment. I'll check with Anuja on any progress but can edit this entry so it doesn't look _too_ quasi-draft haha. 


_Re: Optional: use {bslib} to make cards_
I should have asked you before implementing the cowplot approach, as I cannot tell you how many Google searches I did for inspiration with no luck haha. I just created my own code but love the bslib approach you've gone with as well! Maybe for now for the initial deployment of the new site we can maintain what I've made but look into bslib in the future. I agree the less dependencies the better! For the two people that can be added here, the only requirement for photos is that they're in a 2:3 portrait ratio so they fit with the other photos that others will submit!

_Re: In conclusion_
Totally agree about getting our legacy pages updated. At this point, I've edited majority of them to be .Rmd instead of .md for better spacing and formatting for the site. Content-wise we can definitely work on updating them!

**Comment by jcabercrombie** [comment]
Hi @aclark02-arcus! If you're able to check that my changes meet your expectations and confirm the requested changes have been made, I can finish off this PR and have our website go live. Thank you!

**Review by aclark02-arcus — APPROVED** [review]
LGTM! Great work 👍🏼
I'll let you merge if that's the next step.
